tests/network_test.cpp

```diff
@@ -111,15 +111,19 @@ TEST_F(NetworkTest, transfer_lot_of_blocks) {
   std::vector<std::shared_ptr<DagBlock>> dag_blocks;
   dag_blocks.emplace_back(std::make_shared<DagBlock>(blk));
 
-  // creating lot of non valid blocks just for size
+  // creating lot of blocks just for size
```

Comment:
If change to create valid blocks, in line 134 and 143 should check DAG has correct number of blocks, that's total 101 DAG blocks